Library/Formula/dpkg.rb

```diff
@@ -19,16 +19,23 @@ def install
     inreplace "scripts/Dpkg/Checksums.pm" do |s|
       s.gsub! "md5sum", "md5"
       s.gsub! "sha1sum", "shasum"
-      s.gsub! "sha256sum", "'shasum', '-a', '256'"
+      s.gsub! "sha256sum", "shasum', '-a', '256"
```

Comment:
This is actually correct in this case. The removal of the quote marks here is deliberate.

The original code is: ` program => [ 'sha256sum' ], `, so when we inreplaced as I did originally here we managed to change that to ` program => [ ''shasum', '-a', '256''" ] ` which obviously confused the heck out of the perl-using executables at runtime. We could flip it the other way around, to include the single quotes inside the double quotes around `sha256sum` but I don't think it's any less or more clear one way or another.

As for the spacing - It matches the patch we sent upstream previously. I generally try not to send a patch upstream and then do something else for the sake of not confusing both upstream and us further down the line :smiley_cat:.